src/doc/trpl/error-handling.md

```diff
@@ -279,7 +279,7 @@ fn extension(file_name: &str) -> Option<&str> {
 }
 ```
 
-One other pattern that we find is very common is assigning a default value to
```

Comment:
Alternatively, the sentence could be restructured slightly to avoid the repeated "is":

> One other pattern we commonly find is assigning a default value...